.github/workflows/update-reference-docs.yaml

```diff
@@ -0,0 +1,97 @@
+name: Update Reference Documentation
+
+on:
+#  schedule:
+#    # Run daily at 2 AM UTC
+#    - cron: '0 2 * * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
```

Comment:
consider making this repo_dispatch so we can do post-release events for tool releases without manual action